meta/AMuon/Analytic_Loop_Functions.m

```diff
@@ -0,0 +1,181 @@
+(* ::Package:: *)
+
+(* ::Input:: *)
+(*(*SSF Diagram*)*)
```

Comment:
It seems all the code is commented.
Could you please un-comment it (just the first layer of comments)?

Reply:
Thanks for spotting that.  Should be fixed now.

Reply:
Great, thanks!

Reply:
One wish, though. :)

Could you please add very brief comments to document the function arguments? (Also there are symbols like `\[ScriptD]` where I don't know what they mean.) Maybe it would be best to do this right before the function starts, like so:
~~~.m
(* SSF Diagram

   - mi: fermion mass in the loop???
   - mj: fermion mass in the loop???
   - mF: ...
   - mS: ...
   - SFinleft: ...
   - SFoutleft: ...
   - SFinright: ...
   - SFoutright: ...
 *)
SSFA2L[mi_,mj_,mF_,mS_,SFinleft_,SFoutleft_,SFinright_,SFoutright_]:=Module[{r},
  ...
]
~~~

Reply:
Good idea.  I'll upload those changes.

Reply:
Excellent, thanks!